util/internal/integrationtest/mapper/orm/src/main/java/org/hibernate/search/util/impl/integrationtest/mapper/orm/DatabaseContainer.java

```diff
@@ -284,7 +289,23 @@ HibernateSearchJdbcDatabaseContainer container(Path dockerfile, String name) {
 								.forPort( 8080 )
 								.forStatusCode( 200 ),
 						8080
-				).withCommand( "start-single-node --insecure --store=type=mem,size=0.25 --advertise-addr=localhost" )
+				){
+					@Override
+					protected void containerIsStarted(InspectContainerResponse containerInfo) {
+						super.containerIsStarted( containerInfo );
+						try {
+							// see https://www.cockroachlabs.com/docs/stable/configure-replication-zones#replication-zone-variables
+							// Smaller values can save disk space and improve performance if values are frequently overwritten or for queue-like workloads
+							ScriptUtils.executeDatabaseScript( getDatabaseDelegate(), null,
+									"ALTER DATABASE defaultdb CONFIGURE ZONE USING gc.ttlseconds = 60;\n"
+											+ "ALTER RANGE default CONFIGURE ZONE USING gc.ttlseconds = 60;"
+							);
```

Comment:
Maybe this won't help with our tests much, but that's something to consider for those using outbox-polling with CockroachDB I guess 😃 (just in case you are interested @yrodiere  😃)

Reply:
Makes sense...

I wonder though; those other changes you're making seem like an awful lot of work. Isn't there rather an option like:

```
ALTER DATABASE defaultdb CONFIGURE ZONE USING
schema.dont-be-dumb-i-dont-want-a-distrubuted-database-just-some-fast-stuff.so-make-ddl-fast = true;
```

? :grin:

Reply:
it's hidden somewhere and they don't want to share!!! 🤣